pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -38,6 +46,12 @@ const Name = names.VolumeRestrictions
 const (
 	// ErrReasonDiskConflict is used for NoDiskConflict predicate error.
 	ErrReasonDiskConflict = "node(s) had no available disk"
+	// ErrReasonNodeInfoError is used when errors occur getting node info.
+	ErrReasonNodeInfoError = "error while getting node info"
```

Comment:
we only use constants for the Unschedulable reasons, not for errors. Just put them inline.

Reply:
Done